wrapanapi/systems/ec2.py

```diff
@@ -804,17 +807,26 @@ def object_exists_in_bucket(self, bucket_name, object_key):
         objects = [o for o in bucket.objects.all() if o.key == object_key]
         return any(objects)
 
-    def delete_s3_bucket(self, bucket_name):
-        """TODO: Force delete - delete all objects and then bucket"""
-        bucket = self.s3_connection.Bucket(bucket_name)
-        self.logger.info("Trying to delete bucket '%s'", bucket_name)
+    def delete_s3_buckets(self, bucket_names):
+        """ Deletes specified bucket(s) with keys """
         try:
-            bucket.delete()
-            self.logger.info("Success: bucket '%s' was deleted.", bucket_name)
-            return True
-        except Exception:
-            self.logger.exception("Bucket '%s' deletion failed", bucket_name)
-            return False
+            iter(bucket_names)
```

Comment:
I think it would be better to define the function with `*bucket_names` instead of asking the caller to pass a list.